### ISSUE TITLE
Invalid user/pass in smtpConfig 

### ISSUE THREAD

**Issue by lehdqlsl** [line/abc-user-feedback#244]
``` typescript
export const smtpConfig = registerAs('smtp', () => ({
  use: process.env.SMTP_USE === 'true',
  host: process.env.SMTP_HOST,
  port: +process.env.SMTP_PORT,
  password: process.env.SMTP_USERNAME, <---
  username: process.env.SMTP_PASSWORD, <---
  sender: process.env.SMTP_SENDER,
  baseUrl: process.env.SMTP_BASE_URL,
}));
```

I have found an issue with the SMTP settings. 
The username and password are configured in reverse.

**Comment by h4l-yup** [comment]
@lehdqlsl 
Oh you're right.
I will fix this bug. Thanks

**Comment by h4l-yup** [comment]
It has been fixed on new version(5.2414.51).

Thanks again for your report.
